### PR TITLE
Add a prototype of an integration test of service loading [ECR-3010]

### PR DESCRIPTION
**Pull request by dmitry-timofeev**
## Overview

A simple single IT that successfully loads a fake-service.

---
See: https://jira.bf.local/browse/ECR-3010

### Definition of Done

- [ ] There are no TODOs left in the code
- [ ] Change is covered by automated [tests](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#tests)
- [ ] The [coding guidelines](https://github.com/exonum/exonum-java-binding/blob/master/CONTRIBUTING.md#the-code-style) are followed
- [ ] Public API has Javadoc
- [ ] Method preconditions are checked and documented in the Javadoc of the method
- [ ] Changelog is updated if needed (in case of notable or breaking changes)
- [ ] The continuous integration build passes

### CODE REVIEW

File: exonum-java-binding/core-its/pom.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <dependency>
+      <groupId>com.exonum.binding</groupId>
+      <artifactId>fake-service</artifactId>
+      <version>${project.version}</version>
+      <!-- Depend on "POM" just to ensure fake-service is built before core-its. It won't appear
+           on the classpath as intended. -->
+      <type>pom</type>
+    </dependency>
+  </dependencies>
+</project>
```
Comment (bullet-tooth) [inline exonum-java-binding/core-its/pom.xml]: files should end with an empty line

File: exonum-java-binding/core-its/src/test/java/com/exonum/binding/runtime/Pf4jServiceLoaderIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import java.nio.file.Paths;
+import org.junit.jupiter.api.Test;
+import org.pf4j.DefaultPluginManager;
+import org.pf4j.PluginManager;
+
+class Pf4jServiceLoaderIntegrationTest {
+
+  @Test
+  void loadsFakeService() throws ServiceLoadingException {
+    String fakeServicePath = System.getProperty("exonum.fakeServicePath");
```
Comment (bullet-tooth) [inline exonum-java-binding/core-its/src/test/java/com/exonum/binding/runtime/Pf4jServiceLoaderIntegrationTest.java]: does it work for running the test from Idea directly?

Comment (dmitry-timofeev) [reply exonum-java-binding/core-its/src/test/java/com/exonum/binding/runtime/Pf4jServiceLoaderIntegrationTest.java]: Yes, as core ITs, if it is configured to use Maven configurations.

**Comment by dmitry-timofeev** [comment]
I'd leave it as a prototype, as the overhead of a separate module for such ITs is higher than the benefit (we already have ITs that generate artifacts at runtime using ServiceArtifactBuilder).

**Pull request closed without merging**